models/articles.model.js

```diff
@@ -10,3 +10,29 @@ exports.findArticleById = (article_id) => {
       return result.rows[0];
     });
 };
+
+exports.findArticles = () => {
+  return Promise.all([
+    db.query(
+      "SELECT articles.author, articles.title, articles.article_id, articles.topic, articles.created_at, articles.votes, articles.article_img_url FROM articles ORDER BY created_at DESC"
```

Comment:
I would do these within one query to show off your SQL skills rather than two different queries :)